pyflakes/checker.py

```diff
@@ -600,6 +600,17 @@ def unusedAssignments(self):
                     isinstance(binding, Assignment)):
                 yield name, binding
 
+    def unusedAnnotations(self):
+        """
+        Return a generator for the annotations which have not been used.
+        """
+        for name, binding in self.items():
+            if (not binding.used and
+                    name not in self.globals and
+                    not self.usesLocals and
```

Comment:
I know this is copied from above, but are these still relevant?  does annotating a global do anything? does `locals()` include annotated-only things?

Reply:
Nah I like your point. Seems like it can be simpler, yet still do what we need?